src/vma/dev/qp_mgr_eth_mlx5.cpp

```diff
@@ -224,25 +224,40 @@ inline void qp_mgr_eth_mlx5::set_signal_in_next_send_wqe()
 }
 
 //! Copying src to BlueFlame register buffer by Write Combining cnt WQEBBs
-static inline void copy_bf(uint64_t *bf_dst, uint64_t *src, int cnt)
+inline void qp_mgr_eth_mlx5::copy_bf(uint64_t *bf_dst, uint64_t *src, int cnt)
 {
 	// Avoid using memcpy() to copy to BlueFlame page, since memcpy()
 	// implementations may use move-string-buffer assembler instructions,
 	// which do not guarantee order of copying.
-	while (cnt--) {
-		COPY_64B_NT(bf_dst, src);
+	if (m_b_is_hypervisor) {
```

Comment:
As far as i remember, there were no issues over Windows hyperv,
Assuming I remember correctly, there is no reason to automatically disable it for each VM.